frontend/src/components/LineChart.vue

```diff
@@ -7,7 +7,20 @@
           :max-height="lgAndUp ? '100%' : '20vh'"
           max-width="100%"
           min-width="500px"
+          style="position: relative;"
         >
+        <!-- Add Reset Zoom Icon -->
+        <v-btn
+          class="zoom-button"
+          color="#f4f4f4"
+          outlined
+          @click="resetZoom()"
+          style="position: absolute; top: 110px; right: 10px; z-index: 10;"
+          >
+          <v-icon :icon="mdiMagnifyMinusOutline" class="me-2"></v-icon>
+          RESET ZOOM
+        </v-btn> 
+        <!-- Chart -->
```

Comment:
I like the intent with the reset zoom button!
However, I think it is a bit cumbersome:
<img width="892" alt="image" src="https://github.com/user-attachments/assets/f6374e7e-cc6c-4830-a0b4-3ae860b504b3">

I would make it smaller -- ideally just an icon, no text, add the text in tooltip.
https://vuetifyjs.com/en/components/buttons/#icon

Reply:
maybe @Castronova will have an opinion about the button size...

Reply:
@igarousi here is an example button:
![image](https://github.com/user-attachments/assets/43d47fdb-4174-4f2d-9cf9-76d3ba01e574)

```
<v-tooltip>
        <template #activator="{ props }">
          <v-btn
            v-bind="props"
            color="input"
            @click="resetZoom()"
            style="position: absolute; top: 110px; right: 10px; z-index: 10;"
            :icon="mdiMagnifyMinusOutline"
          >
        </v-btn>
        </template>
        RESET ZOOM
      </v-tooltip> 
```

Reply:
one other thing on this reset button:
there is no longer any `resetZoom()` function in this vue component. So you will have to figure out how to handle that I think...

Reply:
Yes, I just did. Thanks for pointing this out.